front/lib/auth.ts

```diff
@@ -292,6 +292,7 @@ export class Authenticator {
           name: this._workspace.name,
           allowedDomain: this._workspace.allowedDomain || null,
           role: this._role,
+          segmentation: this._workspace.segmentation || null,
```

Comment:
We could do: 
```
segmentation: this.isDustSuperUser() ? this._workspace.segmentation : null,
```

Reply:
Actually the goal is maybe to use it in the product at some point so I'll keep it everywhere